synapse/models/person.py

```diff
@@ -46,6 +47,42 @@ def seedPersonGuidName(self, prop, valu, **props):
             node = self.core.formTufoByProp('ps:person', None, guidname=valu, **props)
         return node
 
+    @modelrev('ps', 201802281621)
+    def _revModl201802281621(self):
+        '''
+        Combine ps:has* into ps:has
+        - Forms a new ps:has node for all of the old ps:has* nodes
+        - Applies the old node's tags to the new node
+        - Deletes the old node
+        - Deletes the syn:tagform nodes for the old form
+        - FIXME do we need to do anything with dark rows?
```

Comment:
@vEpiphyte - Do you think anything else needs to be done here?

Reply:
yes - we will want to add the nodes made by this migration to a dark set by making those rows directly 
see the example here https://github.com/vertexproject/synapse/blob/master/synapse/models/inet.py#L768
you'd want to use updated dprops though 
```
        # mark changed nodes with a dark row...
        dvalu = 'ps:201802281621'
        dprop = '_:dark:syn:modl:rev'
```

Reply:
Done